apps/docs/components/Heading/Heading.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
+
+import React, { type HTMLAttributes } from "react";
+import { usePathname } from "next/navigation";
+import { generateId } from "@/components/Utils/GenerateId/GenerateId";
+
+import "./heading.css";
+
+export type HeadingProps = React.DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement> & {
+    as: "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
+};
+
+export const Heading = ({ as, ...props }: HeadingProps) => {
```

Comment:
The `Title` component does the same thing, so you could use it instead of creating a new one.